README.md

```diff
@@ -167,6 +233,7 @@ Christian
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+<!-- MIT LICENSE -->
 
 ## 📝 License <a name="license"></a>
```

Comment:
- [ ] kindly consider removing the Note on this section to make your readme clean and professional as it was given to you for guidance not to be pushed in your readme 😃